### PR TITLE
Update Postgres CDC  SQL demo default conditions with slot.name

### PR DESCRIPTION
**Pull request by syyfffy**
please merge into master tag and release-2.4 tag

### CODE REVIEW

File: docs/content/connectors/postgres-cdc.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ CREATE TABLE shipments (
   'database-name' = 'postgres',
   'schema-name' = 'public',
   'table-name' = 'shipments',
+  'slot.name' = 'flink',
    -- experimental feature: incremental snapshot (default off)
```
Comment (ruanhang1993) [inline docs/content/connectors/postgres-cdc.md:52]: Please update the sql at line 367 too.

**Review by ruanhang1993 — COMMENTED** [review]
@syyfffy Thanks for changes.
Please raise a new PR for the branch release-2.4 too.

File: docs/content/connectors/postgres-cdc.md (hunk 1 above)
Comment (syyfffy) [reply docs/content/connectors/postgres-cdc.md:52]: i have update the sql at line 367 with commint b915ec49a572a2cd3cfe6fde09f02942f60cf4b5

Comment (syyfffy) [reply docs/content/connectors/postgres-cdc.md:52]: image
as u can see the code for two differecnt line

Comment (syyfffy) [reply docs/content/connectors/postgres-cdc.md:52]: @ruanhang1993  if u have free time ,please review it

**Review by ruanhang1993 — APPROVED** [review]
LGTM.

**Pull request merged**